### PR TITLE
Disable refetchOnMount for paginated query results

### PR DESCRIPTION
**Pull request by dem4ron**
- Fix casing for initial data on student/TracksList
- Manually reset staleTime after mounting
- Better solution
- Disable refetchOnMount generally

### CODE REVIEW

**Review by iHiD — APPROVED** [review]

File: app/javascript/components/student/TracksList.tsx [hunk 1]
```diff
@@ -29,6 +29,7 @@ export default ({
   tagOptions: readonly TagOption[]
   request: Request
 }): JSX.Element => {
+  const [staleTime, setStaleTime] = useState(1000)
```
Comment (iHiD) [inline app/javascript/components/student/TracksList.tsx]: Legacy?

Comment (dem4ron) [reply app/javascript/components/student/TracksList.tsx]: Yes

Comment (ErikSchierboom) [reply app/javascript/components/student/TracksList.tsx]: But this isn't used?

Comment (iHiD) [reply app/javascript/components/student/TracksList.tsx]: No. It's not needed (and not used in the code as far as I can tell).

Comment (dem4ron) [reply app/javascript/components/student/TracksList.tsx]: I used this to try out a concept, and forgot to delete.

**Pull request merged**